mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/proto/GetClusterUsageResponse.java

```diff
@@ -34,7 +33,7 @@ public class GetClusterUsageResponse {
     @Value
     @Builder
     public static class UsageByMachineDefinition {
-        MachineDefinition def;
+        String containerDefinitionId;
```

Comment:
It looks like we are using containerDefintionId and skuId interchangeably. I am wondering if we can conform to one.

Reply:
ContainerDefinitionId was evolved when it was used inside legacy machine definition. How about containerSkuId?